terraform/modules/service/ingest/main.tf

```
@@ -0,0 +1,85 @@
+module "base" {
+  source = "../../service/base"
+
+  service_name = var.service_name
+  cluster_arn  = var.cluster_arn
+
+  container_definitions = [
+    module.nginx_container.container_definition,
+    module.external_api_container.container_definition,
+    module.worker_container.container_definition
+  ]
+
+  desired_task_count = var.desired_task_count
```

I can't find it now, but I'm sure we used to do something like this for the task counts in the API:

```suggestion
  desired_task_count = max(3, var.desired_task_count)
```

with a link to the bit of API Gateway documentation that said "you should run an API in every region".